specification/paths/Invitations.json

```diff
@@ -0,0 +1,126 @@
+              "type": "object",
+              "required": [
+                "self",
+                "first",
+                "last"
+              ],
+              "properties": {
+                "self": {
+                  "type": "string",
+                  "example": "$base_url/invitations?page[number]=3&page[size]=30"
+                },
+                "first": {
+                  "type": "string",
+                  "example": "$base_url/invitations?page[number]=1&page[size]=30"
+                },
+                "prev": {
+                  "type": "string",
+                  "example": "$base_url/invitations?page[number]=2&page[size]=30"
+                },
+                "next": {
+                  "type": "string",
+                  "example": "$base_url/invitations?page[number]=4&page[size]=30"
+                },
+                "last": {
+                  "type": "string",
+                  "example": "$base_url/invitations?page[number]=13&page[size]=30"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "post": {
+    "tags": [
+      "Invitations"
+    ],
+    "summary": "Create a invitation.",
+    "description": "This endpoint creates a new invitation with the posted data. <strong>A invitation will be sent via email.</strong> ",
+    "parameters": [
+      {
+        "in": "body",
+        "name": "password",
+        "description": "The invitation object to be created.",
+        "required": true,
+        "schema": {
+          "type": "object",
+          "properties": {
+            "data": {
+              "type": "object",
+              "required": [
+                "type",
+                "attributes"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "type": {
+                  "$ref": "../types/Invitation.json"
+                },
+                "id": {
+                  "$ref": "../includes/read-only-uuid.json"
+                },
+                "attributes": {
+                  "type": "object",
+                  "required": [
+                    "email"
+                  ],
+                  "additionalProperties": false,
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "example": "James@doe.com"
+                    },
+                    "name": {
+                      "type": "string",
+                      "example": "James Jonson"
+                    },
+                    "message": {
+                      "type": "string",
+                      "example": "Hi James, this is your invitation to join myparcel.com"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    ],
+    "responses": {
+      "202": {
+        "description": "A new invitation token will be sent to the specified user via email."
```

Comment:
Remove the word 'token'? They won't just receive a token, but a link to the backoffice with a token as a query param.